builtin_string.go

```diff
@@ -942,6 +942,8 @@ func (r *Runtime) initString() {
 	o._putProp("trim", r.newNativeFunc(r.stringproto_trim, nil, "trim", nil, 0), true, false, true)
 	o._putProp("trimEnd", r.newNativeFunc(r.stringproto_trimEnd, nil, "trimEnd", nil, 0), true, false, true)
 	o._putProp("trimStart", r.newNativeFunc(r.stringproto_trimStart, nil, "trimStart", nil, 0), true, false, true)
+	o._putProp("trimRight", r.newNativeFunc(r.stringproto_trimEnd, nil, "trimEnd", nil, 0), true, false, true)
+	o._putProp("trimLeft", r.newNativeFunc(r.stringproto_trimStart, nil, "trimStart", nil, 0), true, false, true)
```

Comment:
```suggestion
	trimEnd := r.newNativeFunc(r.stringproto_trimEnd, nil, "trimEnd", nil, 0)
	trimStart := r.newNativeFunc(r.stringproto_trimStart, nil, "trimStart", nil, 0)
	o._putProp("trimEnd", trimEnd, true, false, true)
	o._putProp("trimStart", trimStart, true, false, true)
	o._putProp("trimRight", trimEnd, true, false, true)
	o._putProp("trimLeft", trimStart, true, false, true)
```

Reply:
:facepalm: ... I should've probably figured this out on my own ...